.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+<!-- Thanks for creating this bug report! Please try to provide as much information as possible so we can recreate the bug. Also, try searching GitHub issues to see if someone has previously posted a similar bug. -->
```

Comment:
How can we make it clear that these fields are optional? or are they not optional?

Reply:
They're definitely optional, but helpful for us to debug. Do you think we should explicitly say that they're optional?